tests/unit/core/mock/dispatcher.php

```diff
@@ -82,8 +82,8 @@ public static function create($test, $defaults = true)
 				$mockObject,
 				array(
 					'dispatch' => array(get_called_class(), 'mockDispatch'),
-					'register' => array(get_called_class(), 'mockRegister'),
-					'trigger' => array(get_called_class(), 'mockTrigger'),
+					'addListener' => array(get_called_class(), 'mockRegister'),
+					'triggerEvent' => array(get_called_class(), 'mockTrigger'),
```

Comment:
The interface has dispatch, addListener, and removeListener.  As that's what is being mocked, those are the methods that should be too.